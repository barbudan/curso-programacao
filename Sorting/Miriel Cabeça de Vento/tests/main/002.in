5
mare olori
u i vethed na i onnad
harya mara arya
harya mara lome
naur an edraith ammen