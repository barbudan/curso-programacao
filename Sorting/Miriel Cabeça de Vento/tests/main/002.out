harya mara arya
harya mara lome
mare olori
naur an edraith ammen
u i vethed na i onnad
